tests/api/api.spec.ts

```diff
@@ -1097,7 +1097,7 @@ describe("@dataform/api", () => {
 
       // Make sure it compiles.
       expect(tableNames).includes("example");
-      const example = graph.tables.filter((t: dataform.ITable) => t.name == "example")[0];
+      const example = graph.tables.filter((t: dataform.ITable) => t.name === "example")[0];
```

Comment:
I wonder if there's a rule we can turn on somewhere to require `===` instead of `==`

Reply:
https://palantir.github.io/tslint/rules/triple-equals/ - will enable in a later PR!